### PR TITLE
Update event-4670.md

### LINKED ISSUE #9318: Broken "Left pane"
Broken "Left pane"

ja-jp: "https://docs.microsoft.com/ja-jp/windows/security/threat-protection/auditing/event-4670"
event-4670

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: e5862cc7-3f36-5f2e-9ade-f7441f99f818
* Version Independent ID: b8bd1985-7b5b-ea79-976a-7d96ad69335c
* Content: [Windows イベント転送を使用して侵入検出を支援する (Windows 10) - Windows security](https://docs.microsoft.com/ja-jp/windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection)
* Content Source: [windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/security/threat-protection/use-windows-event-forwarding-to-assist-in-intrusion-detection.md)
* Product: **m365-security**
* Technology: **mde**
* GitHub Login: @DulceMontemayor
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by KarkDenUheldige**
**Description:**

- https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4670

As reported in issue ticket #9318 (**Broken "Left pane"**), the TOC pane overshoots its boundaries and covers the article document partially, in the Japanese version as well as in other automatically machine translated pages.

Thanks to @hyoshioka0128 for reporting the issue appearing on the /ja-jp/ page.

It has been determined that leftover HTML tags is the underlying cause of this issue.

Thanks to @TinaMcN for pointing out the outdated HTML tagging issue.

**Changes proposed:**

- Remove redundant HTML tags and use MarDown formatting wherever possible.
- Use MarkDown Note blob format for Note sections and the Important note.
- Replace outdated and redirected MSDN links with links to their current MS Docs pages.

**Whitespace and formatting corrections:**

- Reduce bullet point list whitespace (blanks) from 3x to 1x (MarkDown default).
- Replace curly quote characters with straight ASCII quote characters.
- Use XML beautify formatting tool to arrange the XML code properly.
- Add MD syntax highlighting tag "xml" to the MD code block fence.

**Ticket closure or reference:**

Closes #9318

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
**Rich Diff view** for easier layout-based review:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9328/files?short_path=4dd4c03#diff-4dd4c038d22cc9c15cabda0cd2ff085a2b7441102713238b0f1f449138f09bcd

**Hide whitespace changes** to slightly reduce the number of default source view lines to review:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9328/files?diff=unified&w=1

**Comment by illfated** [comment]
@JohanFreelancer9 : Kindly perform a copy review. Thank you.

File: windows/security/threat-protection/auditing/event-4670.md [hunk 1]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
-***Event Versions:*** 0.
+**Event Versions:** 0.
 
-***Field Descriptions:***
+**Field Descriptions:**
 
 **Subject:**
 
--   **Security ID** \[Type = SID\]**:** SID of account that requested the “change object’s permissions” operation. Event Viewer automatically tries to resolve SIDs and show the account name. If the SID cannot be resolved, you will see the source data in the event.
+- **Security ID** [Type = SID] : SID of account that requested the "change object’s permissions" operation. Event Viewer automatically tries to resolve SIDs and show the account name. If the SID cannot be resolved, you will see the source data in the event.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:94]:
```suggestion
- **Security ID** [Type = SID]: SID of account that requested the "change object’s permissions" operation. Event Viewer automatically tries to resolve SIDs and shows the account name. If the SID cannot be resolved, you will see the source data in the event.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 2]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
+> [!NOTE]
+> A **security identifier (SID)** is a unique value of variable length used to identify a trustee (security principal). Each account has a unique SID that is issued by an authority, such as an Active Directory domain controller, and stored in a security database. Each time a user logs on, the system retrieves the SID for that user from the database and places it in the access token for that user. The system uses the SID in the access token to identify the user in all subsequent interactions with Windows security. When a SID has been used as the unique identifier for a user or group, it cannot ever be used again to identify another user or group. For more information about SIDs, see [Security identifiers](/windows/access-protection/access-control/security-identifiers).
 
--   **Account Name** \[Type = UnicodeString\]**:** the name of the account that requested the “change object’s permissions” operation.
+- **Account Name** [Type = UnicodeString] : the name of the account that requested the "change object’s permissions" operation.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:99]:
```suggestion
- **Account Name** [Type = UnicodeString]**:** The name of the account that requested the "change object’s permissions" operation.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 3]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
 
--   **Account Domain** \[Type = UnicodeString\]**:** subject’s domain or computer name. Formats vary, and include the following:
+- **Account Domain** [Type = UnicodeString] : subject’s domain or computer name. Formats vary, and include the following:
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:101]:
```suggestion
- **Account Domain** [Type = UnicodeString]**:** Subject’s domain or computer name. Formats vary, and include the following:
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 4]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
+    - Uppercase full domain name: CONTOSO.LOCAL
 
-    -   For some [well-known security principals](https://support.microsoft.com/kb/243330), such as LOCAL SERVICE or ANONYMOUS LOGON, the value of this field is “NT AUTHORITY”.
+    - For some [well-known security principals](https://support.microsoft.com/kb/243330), such as LOCAL SERVICE or ANONYMOUS LOGON, the value of this field is "NT AUTHORITY".
 
-    -   For local user accounts, this field will contain the name of the computer or device that this account belongs to, for example: “Win81”.
+    - For local user accounts, this field will contain the name of the computer or device that this account belongs to, for example: "Win81".
 
--   **Logon ID** \[Type = HexInt64\]**:** hexadecimal value that can help you correlate this event with recent events that might contain the same Logon ID, for example, “[4624](event-4624.md): An account was successfully logged on.”
+- **Logon ID** [Type = HexInt64]**:** hexadecimal value that can help you correlate this event with recent events that might contain the same Logon ID, for example, "[4624](event-4624.md): An account was successfully logged on."
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:113]:
```suggestion
- **Logon ID** [Type = HexInt64]**:** Hexadecimal value that can help you correlate this event with recent events that might contain the same Logon ID, for example, "[4624](event-4624.md): An account was successfully logged on."
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 5]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
 
 **Object**:
 
--   **Object Server** \[Type = UnicodeString\]: has “**Security**” value for this event.
+- **Object Server** [Type = UnicodeString]: has "**Security**" value for this event.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:117]:
```suggestion
- **Object Server** [Type = UnicodeString]**:** Has "**Security**" value for this event.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 6]
```diff
@@ -17,97 +17,106 @@ ms.technology: mde
...
 
--   **Object Type** \[Type = UnicodeString\]: The type of an object that was accessed during the operation.
+- **Object Type** [Type = UnicodeString]: The type of an object that was accessed during the operation.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:119]:
```suggestion
- **Object Type** [Type = UnicodeString]**:** The type of an object that was accessed during the operation.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 7]
```diff
@@ -121,76 +130,77 @@ Before this event can generate, certain ACEs might need to be set in the object
 | Key                     | WaitablePort | Callback             | Semaphore    |
 | Job                     | Port         | FilterConnectionPort | ALPC Port    |
 
--   **Object Name** \[Type = UnicodeString\]: name and other identifying information for the object for which permissions were changed. For example, for a file, the path would be included. For Token objects, this field typically equals “-“.
+- **Object Name** \[Type = UnicodeString\]: name and other identifying information for the object for which permissions were changed. For example, for a file, the path would be included. For Token objects, this field typically equals "-".
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:133]:
```suggestion
- **Object Name** \[Type = UnicodeString\]**:** Name and other identifying information for the object for which permissions were changed. For example, for a file, the path would be included. For Token objects, this field typically equals "-".
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 8]
```diff
@@ -121,76 +130,77 @@ Before this event can generate, certain ACEs might need to be set in the object
...
 
--   **Handle ID** \[Type = Pointer\]: hexadecimal value of a handle to **Object Name**. This field can help you correlate this event with other events that might contain the same Handle ID, for example, “[4663](event-4663.md)(S): An attempt was made to access an object.” This parameter might not be captured in the event, and in that case appears as “0x0”.
+- **Handle ID** \[Type = Pointer\]: hexadecimal value of a handle to **Object Name**. This field can help you correlate this event with other events that might contain the same Handle ID, for example, "[4663](event-4663.md)(S): An attempt was made to access an object." This parameter might not be captured in the event, and in that case appears as "0x0".
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:135]:
```suggestion
- **Handle ID** \[Type = Pointer\]**:** Hexadecimal value of a handle to **Object Name**. This field can help you correlate this event with other events that might contain the same Handle ID, for example, "[4663](event-4663.md)(S): An attempt was made to access an object." This parameter might not be captured in the event, and in that case, appears as "0x0".
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 9]
```diff
@@ -121,76 +130,77 @@ Before this event can generate, certain ACEs might need to be set in the object
...
 
     <img src="images/task-manager.png" alt="Task manager illustration" width="585" height="375" />
 
     If you convert the hexadecimal value to decimal, you can compare it to the values in Task Manager.
 
-    You can also correlate this process ID with a process ID in other events, for example, “[4688](event-4688.md): A new process has been created” **Process Information\\New Process ID**.
+    You can also correlate this process ID with a process ID in other events, for example, "[4688](event-4688.md): A new process has been created" **Process Information\\New Process ID**.
 
--   **Process Name** \[Type = UnicodeString\]**:** full path and the name of the executable for the process.
+- **Process Name** \[Type = UnicodeString\]**:** full path and the name of the executable for the process.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:147]:
```suggestion
- **Process Name** \[Type = UnicodeString\]**:** Full path and the name of the executable for the process.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 10]
```diff
@@ -121,76 +130,77 @@ Before this event can generate, certain ACEs might need to be set in the object
...
 
 **Permissions Change:**
 
--   **Original Security Descriptor** \[Type = UnicodeString\]**:** the old Security Descriptor Definition Language (SDDL) value for the object.
+- **Original Security Descriptor** [Type = UnicodeString]**:** the old Security Descriptor Definition Language (SDDL) value for the object.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:151]:
```suggestion
- **Original Security Descriptor** [Type = UnicodeString]**:** The old Security Descriptor Definition Language (SDDL) value for the object.
```

File: windows/security/threat-protection/auditing/event-4670.md [hunk 11]
```diff
@@ -121,76 +130,77 @@ Before this event can generate, certain ACEs might need to be set in the object
...
 
--   **New Security Descriptor** \[Type = UnicodeString\]**:** the new Security Descriptor Definition Language (SDDL) value for the object.
+- **New Security Descriptor** [Type = UnicodeString]**:** the new Security Descriptor Definition Language (SDDL) value for the object.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/auditing/event-4670.md:153]:
```suggestion
- **New Security Descriptor** [Type = UnicodeString]**:** The new Security Descriptor Definition Language (SDDL) value for the object.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by Kuermel — APPROVED** [review]

**Comment by illfated** [comment]
Corrections applied. Thank you for the copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@illfated , can you please look into conflicts with your PR

**Comment by illfated** [comment]
Sure thing, not a problem. Thank you for notifying me. 🙂 👍🏻

**Pull request closed without merging**